library/kotlin/io/envoyproxy/envoymobile/EngineBuilder.kt

```diff
@@ -218,6 +219,19 @@ open class EngineBuilder(
     return this
   }
 
+  /**
+   * Specify whether to use the getaddr-based system DNS resolver or the c-ares resolver.
+   * Defaults to false.
```

Comment:
Should we add a `note` in here that would explain that specifying that EM should use system resolves causes EM ignore some of the other options exposed by the builder - mainly min DNS refresh and filtering of unroutable families. We could mention that `enableDNSUseSystemResolver` means DNS refresh time equal to 60s and enabled filtering of unroutable families.

Reply:
Yes.